apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -484,6 +502,10 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
     const [selectedPropertiesId, setSelectedPropertiesId] = useState(initialPropertyIds)
     const [isAllPropertiesChecked, setIsAllPropertiesChecked] = useState(initialHasAllProperties)
     const [selectedSectionKeys, setSelectedSectionKeys] = useState(initialSectionIds)
+    const [isConfirmVisible, setIsConfirmVisible] = useState(false)
+
+    const showConfirm = () => setIsConfirmVisible(true)
+    const handleCancel = () => setIsConfirmVisible(false)
```

Comment:
Lets just inline these functions, they are too simple and do not abstract complex code